test/phpunit/PhysicalobjectCsvImporterTest.php

```diff
@@ -423,102 +425,29 @@ public function testDoImportNoFilenameException()
     $importer->doImport();
   }
 
-  public function testDoImportWithUnixNewlines()
+  public function testLoadCsvDataWithOffset()
   {
     $importer = new PhysicalObjectCsvImporter($this->context, $this->vdbcon);
-    $importer->typeIdLookupTable = $this->typeIdLookupTableFixture;
-    $importer->setOrmClasses($this->ormClasses);
-    $importer->setOption('quiet', true);
-
-    $importer->doImport($this->vfs->url().'/unix.csv');
-
-    $this->assertSame(explode(',', $this->csvHeader), $importer->getHeader());
-    $this->assertSame($this->getCsvRowAsAssocArray(), $importer->getRow(0));
-    $this->assertSame(3, $importer->countRowsImported());
-    $this->assertSame(4, $importer->countRowsTotal());
-  }
-
-  public function testDoImportWithWindowsNewlinesAndErrorLog()
-  {
-    $importer = new PhysicalObjectCsvImporter($this->context, $this->vdbcon);
-    $importer->typeIdLookupTable = $this->typeIdLookupTableFixture;
-    $importer->setOrmClasses($this->ormClasses);
-    $importer->setOption('errorLog', $this->vfs->url().'/error.log');
-    $importer->setOption('progressFrequency', 2);
-    $importer->setOption('quiet', true);
-
-    $importer->doImport($this->vfs->url().'/windows.csv');
-
-    $this->assertSame(explode(',', $this->csvHeader), $importer->getHeader());
-    $this->assertSame($this->getCsvRowAsAssocArray(), $importer->getRow(0));
-    $this->assertSame(3, $importer->countRowsImported());
-    $this->assertSame(4, $importer->countRowsTotal());
-  }
-
-  public function testDoImportWithOffset()
-  {
-    $importer = new PhysicalObjectCsvImporter($this->context, $this->vdbcon);
-    $importer->typeIdLookupTable = $this->typeIdLookupTableFixture;
-    $importer->setOrmClasses($this->ormClasses);
     $importer->setOffset(1);
-    $importer->setOption('progressFrequency', 0);
     $importer->setOption('quiet', true);
 
-    $importer->doImport($this->vfs->url().'/unix.csv');
+    $records = $importer->loadCsvData($this->vfs->url().'/unix.csv');
 
     $this->assertSame(explode(',', $this->csvHeader), $importer->getHeader());
-    $this->assertSame($this->getCsvRowAsAssocArray(1), $importer->getRow(1));
-    $this->assertSame(2, $importer->countRowsImported());
-    $this->assertSame(4, $importer->countRowsTotal());
+    $this->assertSame($this->getCsvRowAsAssocArray(1), $records->fetchOne());
+    $this->assertSame(3, $importer->countRowsTotal());
   }
 
-  public function testDoImportWithSetHeader()
+  public function testLoadCsvDatWithSetHeader()
```

Comment:
Nitpick: Is "Dat" short for data or is it something else?

Reply:
Yeah, that's a typo. 😊  It should be "Data". :)